psetv2/blinder.go

```diff
@@ -0,0 +1,561 @@
+package psetv2
```

Comment:
This is a more of a topic of design, one of the main purpose during this implementation, was being able to compile pset package to wasm ie. having a way to detach all the CGO operation out of the package.

We may need to have a dedicated pkg for blinder role maybe?

Reply:
only option is to move it to other pkg but then we would also need to export all fields of Pset(Globa, Input, Output)

Reply:
what if we do like the signer role, where we expect via methods to pass stuff to be added ie. AddSignature?

Reply:
but I think is fine to export those fields maybe, for easy access?